docs/manage/scheduled-views/scheduled-views-best-practices.md

```diff
@@ -222,6 +222,8 @@ _view=nice_view_man | timeslice 1d | sum(_count) by _timeslice, status_code
 
 ## FAQ
 
-### Upgrade your scheduled views to avoid negative count results
+### How to avoid negative count results?
```

Comment:
I'd change it to `How do I avoid negative count results?`
Or remove the question mark (`How to avoid negative count results`)